cmd/eval.go

```diff
@@ -88,3 +36,52 @@ func validateFile(file []byte) error {
 	var holder interface{}
 	return yaml.Unmarshal(file, &holder)
 }
+
+var (
+	evalCmd = &cobra.Command{
+		Use:   "eval [file]",
+		Short: "Evaluate the combinations for a file at the given path",
+		Long: `
+Evaluate the combinations for a file at the given path. The file provided must be valid YAML.
+
+Note: the combo binary requires the --replacement flag to be explicitly set.
+
+The replacements flag allows users to specify a series of key value pairs in the form of KEY=VALUES.
+
+Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
+	`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			replacements, _ := cmd.Flags().GetStringToString("replacements")
+
+			file, err := os.ReadFile(args[0])
+			if err != nil {
+				return err
+			}
+
+			if err := validateFile(file); err != nil {
```

Comment:
Any gauge on the UX for an existing, empty file here? Produce a warning message, or an error message indicating that there's no substitution to be performed?

Reply:
I'm fine with marking this out-of-scope for this PR if you want to open a follow-up issue/PR/etc. to handle that edge case.

Reply:
I like this, let me change up that valdiateFile function to do this.

Reply:
```shell
go run main.go eval -r "TEST=1" test.yaml
Error: failed to validate file specified: empty file
Usage:
  combo eval [file] [flags]

Flags:
  -h, --help                          help for eval
  -r, --replacements stringToString   Key value pair of comma delimited values. Example: 'NAMESPACE=foo,bar' (default [])

failed to validate file specified: empty file
exit status 1
```

Reply:
Cool, that works for me for now. We'll want to iron out that UX eventually in the context of CI pipelines running the eval sub-command.